src/core/lv_obj_pos.c

```diff
@@ -854,6 +855,15 @@ void lv_obj_invalidate(const lv_obj_t * obj)
     lv_obj_invalidate_area(obj, &obj_coords);
 }
 
+bool is_transformed(const lv_obj_t * obj)
+{
+    while(obj) {
+        if(obj->spec_attr && obj->spec_attr->layer_type == LV_LAYER_TYPE_TRANSFORM) return true;
+        obj = obj->parent;
```

Comment:
Unreachable code due to previous return at 861. Please remove

Reply:
the purpose of this code is to check object and its parent has transform property.

Reply:
ack